ArcadiaParties-SPA/src/app/user/user-identity/user-identity.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component } from '@angular/core';
+import { Store, select } from '@ngrx/store';
+import { Observable } from 'rxjs';
+
+import { loadUser } from '../actions/actions';
+import { State } from '../../reducers/reducers';
```

Comment:
You should not use things from app module in user module. User module should know nothing about app module or some shared things should be moved to shared folder like `core`.

Reply:
Now using UserState from user module.